Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -4,6 +4,7 @@
 
 import Common
 import Foundation
+import Ecosia
```

Comment:
I see you're following the pattern to add an Ecosia comment like:
```suggestion
// Ecosia: Import Ecosia Framework
import Ecosia
```

To be honest, since "Ecosia" is part of the code, I think this is straight-forward even without the comment and would be up for removing that comment from all `import Ecosia` lines. What do you think?

Reply:
I'm in general up for that 👍 . 
This would be the exception compared to wherever we add the `// Ecosia` comment in Firefox code.
If you're happy, I can go ahead and simplify

Reply:
Agreed!